streamz/core.py

```diff
@@ -307,16 +307,18 @@ def wrapped(*args, **kwargs):
     @classmethod
     def register_plugin_entry_point(cls, entry_point, modifier=identity):
         def stub(*args, **kwargs):
+            """ Entrypoints-based streamz plugin. Will be loaded on first call. """
             node = entry_point.load()
             if not issubclass(node, Stream):
                 raise TypeError(
                     f"Error loading {entry_point.name} "
                     f"from module {entry_point.module_name}: "
                     f"{node.__class__.__name__} must be a subclass of Stream"
                 )
-            cls.register_api(
-                modifier=modifier, attribute_name=entry_point.name
-            )(node)
+            if getattr(cls, entry_point.name).__name__ == "stub":
```

Comment:
I don't *think* this can be False, but it doesn't hurt to check.

Reply:
It is false when the plugin class returned from `entry_point.load()` is decorated with `@Stream.register_api`. Then it is registered right away when loaded, and so at that moment stub is overwritten with the actual class.